burger/style/my-theme.css

```diff
@@ -0,0 +1,8 @@
+.my-theme {
+    --font-color-orange: #e9072d;
+    --font-color-orange2: #f5074f;
```

Comment:
don't use `orange` in your variable name if in other theme it's not orange

Reply:
fixed all changes